### ISSUE TITLE
Fix iOS 10 warnings for language code

### ISSUE THREAD

**Issue by msmollin** [mapzen/on-the-road_ios#37]
- WARN  | [Mapzen-ios-sdk/Core] xcodebuild:  OnTheRoad/OTRRouting/OTRRoutingController.m:65:56: warning: 'languageCode' is only available on iOS 10.0 or newer [-Wunguarded-availability]

**Comment by msmollin** [comment]
Full warnings here:
```
 -> OnTheRoad (1.0.0)
    - WARN  | xcodebuild:  OnTheRoad/OTRRouting/OTRRoutingController.m:62:85: warning: 'languageCode' is only available on iOS 10.0 or newer [-Wunguarded-availability]
    - NOTE  | xcodebuild:  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator11.1.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSLocale.h:36:38: note: 'languageCode' has been explicitly marked partial here
    - NOTE  | xcodebuild:  OnTheRoad/OTRRouting/OTRRoutingController.m:62:85: note: enclose 'languageCode' in an @available check to silence this warning
    - WARN  | xcodebuild:  OnTheRoad/OTRRouting/OTRRoutingController.m:65:56: warning: 'languageCode' is only available on iOS 10.0 or newer [-Wunguarded-availability]
    - NOTE  | xcodebuild:  OnTheRoad/OTRRouting/OTRRoutingController.m:65:56: note: enclose 'languageCode' in an @available check to silence this warning
```

**Comment by msmollin** [comment]
Fixed by #38
